frontend/test/metabase/scenarios/custom-column/reproductions/27745-cc-numeric-missing-summarize.cy.spec.js

```diff
@@ -0,0 +1,78 @@
+import {
+  restore,
+  startNewQuestion,
+  enterCustomColumnDetails,
+  visualize,
+  popover,
+  queryQADB,
+} from "__support__/e2e/helpers";
+
+const createTableQueries = {
+  postgres: `
+    DROP TABLE IF EXISTS colors;
+
+    CREATE TABLE colors (
+      id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+      color VARCHAR ( 255 ) UNIQUE NOT NULL
+    );
+
+    INSERT INTO colors (color) VALUES ('red'), ('green'), ('blue');
+  `,
+  mysql: `
+    DROP TABLE IF EXISTS colors;
+
+    CREATE TABLE colors (
+      id INTEGER PRIMARY KEY AUTO_INCREMENT,
+      color VARCHAR ( 255 ) UNIQUE NOT NULL
+    );
+
+    INSERT INTO colors (color) VALUES ('red'), ('green'), ('blue');
+  `,
+};
+
+const snapshotMap = {
+  postgres: "postgres-12",
+  mysql: "mysql-8",
+};
+
+["postgres", "mysql"].forEach(dialect => {
```

Comment:
Not sure that we need to repro this for both databases, but it definitely doesn't hurt.